.github/workflows/ci.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       with:
         java-version: '11'
         distribution: 'temurin'
+    - name: Check with Gradle
+      uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
+      with:
+        arguments: check
```

Comment:
Why did you add `check`? The next step is `build`, which run spotless as well. In my opinion we do not need `check`.

Reply:
Removed